.github/workflows/ci.yml

```diff
@@ -73,6 +77,7 @@ jobs:
     strategy:
       matrix:
         node-version: [20]
+        arch: ['NewArch', 'OldArch']
```

Comment:
to run check for new & old archs simultaneously

<img width="404" alt="Screenshot 2024-06-14 at 7 29 36 PM" src="https://github.com/react-native-menu/menu/assets/1898138/232fe9f4-9904-4a57-b7b7-f7f15e5c278d">